tests/test_packages/test_skills/test_erc1155.py

```diff
@@ -51,27 +58,82 @@ def test_generic(self, pytestconfig):
         self.force_set_config(setting_path, ledger_apis)
         self.add_item("connection", "fetchai/oef:0.2.0")
         self.set_config("agent.default_connection", "fetchai/oef:0.2.0")
-        self.add_item("skill", "fetchai/erc1155_deploy:0.2.0")
+        self.set_config("agent.default_ledger", "ethereum")
+        self.add_item("skill", "fetchai/erc1155_deploy:0.3.0")
+        self.generate_private_key("ethereum")
+        self.add_private_key("ethereum", "eth_private_key.txt")
+        self.replace_private_key_in_file(
+            FUNDED_ETH_PRIVATE_KEY_1, "eth_private_key.txt"
+        )
+        # stdout = self.get_wealth("ethereum")
+        # if int(stdout) < 100000000000000000:
+        #     pytest.skip("The agent needs more funds for the test to pass.")
```

Comment:
Note, this is why getting the wealth is important. We don't want to run the test if there are insufficient funds on the key.